lib/api/create_constraint.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  class CreateConstraint
+    def matches?(request)
+      Hash(JSON.parse(request.body.read)).fetch("action", "create").in?(%w(create add))
```

Comment:
@imtayadeway Why do you need the `Hash()` call here? `parse` defaults to `Hash`, no?

Reply:
It does, unless the body is empty:

```ruby
JSON.parse("null") # => nil
```